mfr/extensions/papaya/static/data/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
Not sure what you're getting at here. In any case, the .gitignore was for my convenience while developing. Removing it.

Reply:
For example, here is a directory `/repo/level_1/level_2/level_3/`. If you want to ignore all files in `level_3/`, instead of having a `level3/.gitignore` you can use a `level2/.gitignore` or even a repo level one that ignores everything in the directory `level3/`.

Reply:
Travis failed, which reminded me of why I had the .gitignore. In addition to it being handy during development, I also need it to create and check in the empty directory mfr/extensions/papaya/static/data/

Reply:
👍 I will locally tested it again today before sending this to PCR.